### PR TITLE
Add linkedIn share-link

### PR DESCRIPTION
**Pull request by beaubbe**

### CODE REVIEW

File: workspace/utilities/lib/share-link.xsl [hunk 1]
```diff
@@ -45,7 +45,7 @@
 		
 		<xsl:variable name="computed-attr">
 			<xsl:copy-of select="$attr" />
-			<xsl:add data-mailto="{$computed-url}" />
+			<add data-mailto="{$computed-url}" />
```
Comment (nitriques) [inline workspace/utilities/lib/share-link.xsl]: This should have been

1. A seperate PR
1. a `set` ?

Comment (PascalPiche) [reply workspace/utilities/lib/share-link.xsl]: Change add to set for sure

Comment (nitriques) [reply workspace/utilities/lib/share-link.xsl]: Ping @beaubbe

File: workspace/utilities/lib/share-link.xsl [hunk 2]
```diff
@@ -93,4 +93,31 @@
 			<xsl:with-param name="content" select="$content"/>
 		</xsl:call-template>
 	</xsl:template>
+	
+	<xsl:template name="share-link-linkedIn">
```
Comment (nitriques) [inline workspace/utilities/lib/share-link.xsl]: All of our templates are kebab-case. Why is this one not `share-link-linkedin` or `share-link-linked-in` ?

**Pull request merged**